charts/ingress-nginx/templates/_helpers.tpl

```diff
@@ -30,14 +30,6 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- end -}}
 {{- end -}}
 
-{{/*
-Create a default fully qualified controller name.
-We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
-*/}}
-{{- define "ingress-nginx.controller.fullname" -}}
```

Comment:
Why don't we leave this in and include the hardcoded name? This way we have a single place it's defined instead of all over the place.

Reply:
OK